counterpartylib/protocol_changes.json

```diff
@@ -64,5 +64,11 @@
 			"minimum_version_minor": 54,
 			"minimum_version_revision": 0,
 			"block_index": 401278
+		},
+	"bytespersigop":      {
+			"minimum_version_major": 9,
+			"minimum_version_minor": 54,
+			"minimum_version_revision": 0,
+			"block_index": 413083
```

Comment:
make this 500k so that we can safely merge into `develop` and commit with the next release, as this problem has an impact but not a huge one